### ISSUE TITLE
Release infrastructure

### ISSUE THREAD

**Issue by lluchs** [openclonk/openclonk#76]
We have working snapshots for Windows and Linux now. We'll need to extend this to proper releases. Building a release will probably be triggered by pushing a tag.

 - [ ] Implement upload endpoint for the release.
     - [ ] Update download page
 - [ ] Build and upload **Linux** releases
    - These will look exactly like the snapshots (AppImage).
  - [ ] Build and upload **Windows** releases.
     - Installer instead of zip archive.
   - [ ] Source download?
     - Maybe just link to github.

**Comment by walachey** [comment]
>Building a release will probably be triggered by pushing a tag.

Don't forget that we also need a way to trigger a dry release for testing (possibly also for an old revision for bisecting problems?). Either without using git or by having many RC/testing tags in the repos.

**Comment by lluchs** [comment]
Right. Our existing release tags look like `vX.Y` or `vX.Y.Z`, so we could have pre-release tags looking like `vX.Y-abc`, e.g. `v9.0-rc1`. I think we had less than five dry releases for 8.0, so the additional tags are probably fine. In any case, we can clean up and remove the pre-release tags after a successful release.

I think old revisions are also trivially covered by this scheme, as you can create and push tags for any revision.

**Comment by probonopd** [comment]
Can the AppImage snapshots be downloaded somewhere? Thanks.

**Comment by isilkor** [comment]
I don't believe old ones are kept, but the current snapshot is available 
at <https://www.openclonk.org/snapshots/>.
